scripts/rdf-generator/generate.py

```diff
@@ -82,4 +82,8 @@ def generate_crf(n, s, m, p):
         for line in file:
             (n, s, m, p) = line.rstrip().split(",")
             crf = generate_crf(n, s, m, p)
+<<<<<<< HEAD
+            print(crf.serialize(format="turtle").decode("UTF-8"))
+=======
             crf.serialize(f"{out_path}/{n.zfill(5)}.ttl", format="turtle")
+>>>>>>> upstream/develop
```

Comment:
Some remnants of a conflict. I can take care of this when I update the rdf-generator code to reflect the new changes.